util/multicluster/multicluster.go

```diff
@@ -0,0 +1,11 @@
+package multicluster
+
+import (
+	"os"
+)
+
+// IsEnabled is a flag for whether the new multi-cluster feature is
+// enabled
+func IsEnabled() bool {
```

Comment:
Can this please be “ARGO_MULTICLUSTER_ENABLED=true”

Reply:
Will do